content/updates/2016-12-20-ew-sandbox-deprecation.md

```diff
@@ -0,0 +1,19 @@
++++
+date = "2016-12-20"
+title = "East/West Sandbox Deprecation Notice"
++++
+
+The original cloud.gov in AWS East/West is now officially deprecated and will be retired soon. We will be in touch to assist with migration of Prototyping, FISMA Moderate, and FISMA Low organizations to the new environment shortly. 
```

Comment:
Maybe `The original cloud.gov environment in AWS East/West`? I know what you mean, and most users probably will, but wouldn't want anyone worrying unnecessarily that the product has been replaced in a way that invalidates parts of IAAs or whatever that we have in place.